pkg/crc/cluster/cluster.go

```diff
@@ -310,7 +310,7 @@ func WaitForRequestHeaderClientCaFile(sshRunner *ssh.Runner) error {
 }
 
 func WaitForAPIServer(ocConfig oc.Config) error {
-	logging.Debugf("Waiting for apiserver availability")
+	logging.Info("Waiting for kube-apiserver availability(takes around 2m)...")
```

Comment:
A missing space before the (. 
Also, here we use [] not (): https://github.com/code-ready/crc/blob/master/pkg/crc/machine/start.go#L393